### PR TITLE
Use more specific urls for scriptSrc needed for ANDI 508 testing tool

### PR DESCRIPTION
**Pull request by Matthew-Grayson**
#  #

## 💭 Motivation and context ##
I used improper syntax in the last PR: `https://www.ssa.gov/accessibility/andi` which does not cover all scripts from this url such as `https://www.ssa.gov/accessibility/andi/fandi.js` and `https://www.ssa.gov/accessibility/andi/andi.js`.
This PR hard-codes these two scripts as well as `https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js`.
These are the urls for these scripts, exactly as they are seen in the browser console's network tab.

## ✅ Pre-approval checklist ##

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - *eschew scope creep!*
- [ ] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by rapidray12 — APPROVED** [review]
Simple enough

**Review by cduhn17 — APPROVED** [review]
LGTM
